W-21551268-ch-failed-automated-updates-kt (#873)
mulesoft/docs-runtime-manager: cloudhub/modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

[IMPORTANT]

> Using a runtime version that is more than three months old can introduce security vulnerabilities. If your application doesn't receive any patches for two consecutive months, the application is stopped and force-updated to the latest patch version. Applications that fail to start require your action to address patches. 

Reviewer: ```suggestion
To avoid introducing security vulnerabilities, if your application doesn't receive any patches for two consecutive months, the application is stopped and force-updated to the latest patch version. Applications that fail to start require your action to address patches. 
```
